File: scratch/fix_ui_imports.py
import os
import re

ui_dir = r"d:\GitHub\aulavirtual\src\features\web\landing\components\ui"

for filename in os.listdir(ui_dir):
    if filename.endswith(".tsx"):
        filepath = os.path.join(ui_dir, filename)
        with open(filepath, 'r', encoding='utf-8') as f:
            content = f.read()
        
        # Replace "@/components/ui/XXX" with "./XXX"
        # and "@components/ui/XXX" with "./XXX"
        new_content = re.sub(r'from "@/?components/ui/', 'from "./', content)
        
        if content != new_content:
            with open(filepath, 'w', encoding='utf-8') as f:
                f.write(new_content)
            print(f"Updated {filename}")

File: scratch/fix_ui_imports.js
const fs = require('fs');
const path = require('path');

const uiDir = 'd:/GitHub/aulavirtual/src/features/web/landing/components/ui';

fs.readdirSync(uiDir).forEach(file => {
    if (file.endsWith('.tsx')) {
        const filePath = path.join(uiDir, file);
        let content = fs.readFileSync(filePath, 'utf8');
        
        const newContent = content.replace(/from "@\/?components\/ui\//g, 'from "./');
        
        if (content !== newContent) {
            fs.writeFileSync(filePath, newContent, 'utf8');
            console.log(`Updated ${file}`);
        }
    }
});
